dpdbaltics.php

```diff
@@ -1267,6 +1273,7 @@ public function hookActionOrderGridDefinitionModifier(array $params)
 
         $definition = $params['definition'];
 
+        if (!(bool) Configuration::get(Config::HIDE_ORDERS_LABEL_PRINT_BUTTON)) {
```

Comment:
Allowing bulk actions, but removed singular?

Reply:
We got approvement that we hide button only and keeping bulk actions